packages/cf.js/src/types/events.ts

```diff
@@ -30,9 +31,15 @@ export interface UpdateStateEventData extends AppEventData {
   action?: AppAction;
 }
 
+export interface CreateMultisigEventData {
+  owners: Address[];
+  multisigAddress: Address;
+}
+
 export interface ErrorEventData {
   errorName: string;
   message?: string;
+  appInstanceId?: string;
   extra?: { [k: string]: string | number | boolean | object };
```

Comment:
I think we should enforce these fields to exist and not use `?`

Reply:
We should. For now we don't know what errors we have though. 

I could write a couple and strictly type this for now, until we think more about the errors that the Node can return.

Reply:
What does an `extra` field do?

I'd imagine if the `errorName` was an enum then the necessary information could be conveyed to the client through just the `message` right?

Reply:
Maybe, but there could still be extra (structured) data that's important for CFjs. I could remove it & re-add once it becomes clear